### PR TITLE
Adds a nurse dress to loadout, no I'm not sorry

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

adds a nurse dress ingame and to loadout options. it's all modular, too, which means I created the modular folders, so yay?

## Why It's Good For The Game

I wasn't going to do it but people said 'yeah do it' and I'm horny enough to oblige. also sets up framework for modular stuff w/e lol

## Changelog
```changelog
add: adds nurse dress to loadout
```

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
pics (idk why they're so blurryl mao)

image
image
image

the last one isn't a good picture b/c tail but whatever

File: cev_eris.dme [hunk 1]
```diff
@@ -2810,6 +2810,7 @@
 #include "zzzz_modular_occulus\code\modules\economy\vending.dm"
 #include "zzzz_modular_occulus\code\modules\game\Objects\Items\oddities.dm"
 #include "zzzz_modular_occulus\code\modules\game\Objects\Items\Devices\artifacts.dm"
+#include "zzzz_modular_occulus\code\modules\loadout\lists\uniforms.dm"
```
Comment (Jamini) [inline cev_eris.dme:2813]: This file needs to be added to your commit.

File: cev_eris.dme [hunk 2]
```diff
@@ -2823,4 +2824,5 @@
 #include "zzzz_modular_occulus\code\modules\sprite_accessories\_accessory_hair.dm"
 #include "zzzz_modular_occulus\code\modules\sprite_accessories\_accessory_tesh.dm"
 #include "zzzz_modular_occulus\maps\submaps\deepmaint_rooms\normal\normal_rooms.dm"
+#include "zzzz_modular_occulus\modules\clothing\under\medsci.dm"
```
Comment (Jamini) [inline cev_eris.dme:2827]: This file needs to be added to your commit. It's missing.

File: code/modules/clothing/under/jobs/medsci.dm [hunk 3]
```diff
@@ -120,8 +120,8 @@
 
 
 /obj/item/clothing/under/rank/paramedic
-	desc = "Jumpsuit of Moebius Medical paramedics. It's made with reinforced fiber to offer more protection for recovery operators."
-	name = "moebius paramedic's jumpsuit"
+	desc = "Jumpsuit of NanoTrasen medical paramedics. It's made with reinforced fiber to offer more protection for recovery operators."
+	name = "NanoTrasen paramedic's jumpsuit"
```
Comment (Jamini) [inline code/modules/clothing/under/jobs/medsci.dm:124]: This change should be in your modular file. If it is, this file needs to be reverted back to the master file.

**Review by Jamini — CHANGES_REQUESTED** [review]
Please don't edit base files. any new icons should not go in 

icons/inventory/uniform/icon.dmi
icons/inventory/uniform/mob.dmi
icons/inventory/unfirm/mob_fem.dmi

They should go into a new file in 

zzzz_modular_occulus/icons/inventory/uniform/icon.dmi
zzzz_modular_occulus/icons/inventory/uniform/mob.dmi
zzzz_modular_occulus/icons/inventory/uniform/mob_fem.dmi

**Comment by bearrrrrrrr** [comment]
i know what went wrong
committed tracked files and dropped untracked, fml
fixing

**Pull request closed without merging**